### PR TITLE
Adding showing email of the author of deck 

### PR DESCRIPTION
**Pull request by ArekArek**
This PR is like yesterday but now there are no bugs with git, so it could be merged.

### CODE REVIEW

**Comment by VanDavv** [comment]
Representation have creatorEmail in it, dont change DB model! Just make representation at resource endpoind. DB layer shouldn't be aware of requirements in app layer

File: backend/app/src/main/java/com/bls/patronage/api/DeckRepresentation.java [hunk 1]
```diff
@@ -13,13 +13,15 @@
...
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class DeckRepresentation implements DbMappable<Deck> {
-    private UUID id;
     @NotEmpty
     @Length(min = 1, max = 50)
     final private String name;
     @NotNull
     final private Boolean publicVisible;
+
+    private UUID id;
```
Comment (mlotysz) [inline backend/app/src/main/java/com/bls/patronage/api/DeckRepresentation.java]: Should be on top class - it's most important thing...

File: backend/db/src/main/java/com/bls/patronage/db/dao/DeckDAO.java [hunk 2]
```diff
@@ -39,24 +39,34 @@
...
+            "where decks.id = :id")
     abstract Deck getDeck(@Bind("id") UUID id);
 
     @SqlQuery("select userId from usersDecks where deckId = :id")
     abstract String getDeckUserId(@Bind("id") UUID id);
 
-    @SqlQuery("select id, name, public from decks where name like :name and public='true'")
+    @SqlQuery("select decks.id, decks.name, decks.public, users.email from "+
+            "decks inner join usersDecks on usersDecks.deckId = decks.id "+
+            "inner join users on users.id = usersDecks.userId "+
```
Comment (mlotysz) [inline backend/db/src/main/java/com/bls/patronage/db/dao/DeckDAO.java]: Note: `inner` is default type of join - you could leave `join` only.

File: backend/db/src/main/java/com/bls/patronage/db/model/Deck.java [hunk 3]
```diff
@@ -53,6 +58,10 @@ public void setIsPublic(Boolean isPublic) {
         this.isPublic = isPublic;
     }
 
+    public String getCreatorEmail(){ return creatorEmail;}
+
+    public void setCreatorEmail(String creatorEmail){this.creatorEmail=creatorEmail;}
```
Comment (mlotysz) [inline backend/db/src/main/java/com/bls/patronage/db/model/Deck.java]: Fix formatting.

**Comment by mlotysz** [comment]
Let's discuss this PR and solutions on our meeting.

**Comment by mlotysz** [comment]
Closed - no activity.

**Pull request closed without merging**